aiohttp/connector.py

```diff
@@ -72,6 +71,8 @@
     from .client_reqrep import ConnectionKey  # noqa
     from .tracing import Trace  # noqa
 
+_ListOfFutures = List['asyncio.Future[Optional[Exception]]']
```

Comment:
Is it a list of futures that contains None or Exception as a **value** or list of futures with `fut.set_result(None)` or `fut.set_exception(exc)`?
If later -- please use `Future[None]`.
I believe we never return an exception but set it explicitly.